### ISSUE TITLE
"The function common-lisp:nil is undefined" when generating doc

### ISSUE THREAD

**Issue by zmyrgel** [stumpwm/stumpwm#639]
I'm getting following error when I tried to bump OpenBSD port version from 18.05 to 18.11.

```
Unhandled undefined-function in thread #<sb-thread:thread "main thread" running
                                          {10005C85B3}>:
  The function common-lisp:nil is undefined.

Backtrace for: #<SB-THREAD:THREAD "main thread" RUNNING {10005C85B3}>
0: (SB-DEBUG::DEBUGGER-DISABLED-HOOK #<UNDEFINED-FUNCTION NIL {1002CBE633}> #<unused argument> :QUIT T)
1: (SB-DEBUG::RUN-HOOK *INVOKE-DEBUGGER-HOOK* #<UNDEFINED-FUNCTION NIL {1002CBE633}>)
2: (INVOKE-DEBUGGER #<UNDEFINED-FUNCTION NIL {1002CBE633}>)
3: (ERROR #<UNDEFINED-FUNCTION NIL {1002CBE633}>)
4: (SB-KERNEL:WITH-SIMPLE-CONDITION-RESTARTS ERROR NIL UNDEFINED-FUNCTION :NAME NIL)
5: (SB-IMPL::RETRY-%COERCE-NAME-TO-FUN NIL NIL)
6: (STUMPWM::GENERATE-FUNCTION-DOC #<SB-SYS:FD-STREAM for "file /usr/ports/pobj/stumpwm-18.11/stumpwm-18.11/stumpwm.texi" {10029F6893}> "@@@ read-line-from-sysfs")
7: (STUMPWM::GENERATE-MANUAL :IN #P"stumpwm.texi.in" :OUT #P"stumpwm.texi")
8: (SB-INT:SIMPLE-EVAL-IN-LEXENV (STUMPWM::GENERATE-MANUAL) #<NULL-LEXENV>)
9: (SB-IMPL::SIMPLE-EVAL-PROGN-BODY ((STUMPWM::GENERATE-MANUAL) (QUIT)) #<NULL-LEXENV>)
10: (SB-INT:SIMPLE-EVAL-IN-LEXENV (PROGN (STUMPWM::GENERATE-MANUAL) (QUIT)) #<NULL-LEXENV>)
11: (EVAL (PROGN (STUMPWM::GENERATE-MANUAL) (QUIT)))
12: (SB-IMPL::PROCESS-EVAL/LOAD-OPTIONS ((:EVAL . "(progn (load \"load-stumpwm.lisp\") (load \"manual.lisp\"))") (:EVAL . "(progn (stumpwm::generate-manual) (sb-ext:quit))")))
13: (SB-IMPL::TOPLEVEL-INIT)
14: ((FLET SB-UNIX::BODY :IN SAVE-LISP-AND-DIE))
15: ((FLET "WITHOUT-INTERRUPTS-BODY-14" :IN SAVE-LISP-AND-DIE))
16: ((LABELS SB-IMPL::RESTART-LISP :IN SAVE-LISP-AND-DIE))

unhandled condition in --disable-debugger mode, quitting
*** Error 1 in /usr/ports/pobj/stumpwm-18.11/stumpwm-18.11 (Makefile:24 'stumpwm.texi')
*** Error 1 in . (/usr/ports/infrastructure/mk/bsd.port.mk:2781 '/usr/ports/pobj/stumpwm-18.11/.build_done')
*** Error 1 in /usr/ports/x11/stumpwm (/usr/ports/infrastructure/mk/bsd.port.mk:2447 'all')
```

I'm using threaded SBCL version 1.5.2. Compiling the older 18.05 version works just fine and I've only bumped the stumpwm version and I get the above issue. Any idea what's causing it?

**Comment by PuercoPop** [comment]
> Any idea what's causing it?

Yeah, the manual is looking up the function definition of a function that is not defined when building StumpWM on OpenBSD. afaik that function is `read-line-from-sysfs` and the issue should be fixed on master. See https://github.com/stumpwm/stumpwm/pull/543

**Comment by zmyrgel** [comment]
Right you are, backporting the doc fix pointed by #543 fixed this as well.
